distrib-dl

```diff
@@ -223,6 +227,22 @@ function download_gentoo_live() {
     cd "$download_dir/gentoo/" && sha512sum --ignore-missing -c <(cat ${gentoo_sums_filename}) && cd -
 }
 
+download_opensuse_tumbleweed() {
+    # https://www.opensuse.org/#Tumbleweed
+    opensuse_base_url="http://opensuse.mirrors.theom.nz/${opensuse_release}/iso/"
+    opensuse_iso_filename="${opensuse_image}.iso"
+    opensuse_sums_filename="${opensuse_iso_filename}.sha256"
+    if [[ ! -d "$download_dir/opensuse/" ]]; then mkdir -p "$download_dir/opensuse/"; fi
+    echo "[distrib-dl] INFO: Downloading Opensuse Tumbleweed keyring, checksums and signature..."
+    gpg --keyserver hkps://keyserver.ubuntu.com --recv-keys "$opensuse_gpg_fingerprint"
+    wget -N -nv --show-progress -P "$download_dir/opensuse/" "${opensuse_base_url}${opensuse_sums_filename}"
+    gpg --verify "opensuse/${opensuse_sums_filename}"
+    echo "[distrib-dl] INFO: Downloading OpenSUSE Tumbleweed ISO image..."
+    wget --continue -N -nv --show-progress -P "$download_dir/opensuse/" "${opensuse_base_url}/${opensuse_iso_filename}"
```

Comment:
`wget $wget_opts --timestamping --show-progress --directory-prefix="$download_dir/opensuse/" "${opensuse_base_url}/${opensuse_iso_filename}"` for consistency with other distributions in the script (I changed the wget logic some time ago)

Reply:
OpenSUSE should also be added to the help text (`$usage` variable) and the README